Light.json

```diff
@@ -57774,6 +57774,1068 @@
+      "legacy": false
+    },
+    {
+      "id": 6932,
+      "gempId": "216_29",
+      "side": "Light",
+      "rarity": "U",
+      "set": "216",
+      "printings": [
+        {
+          "set": "216"
+        }
+      ],
+      "front": {
+        "title": "•Grogu",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual16-Light/large/grogu.gif",
+        "type": "Character",
+        "subType": "Alien",
+        "uniqueness": "*",
+        "destiny": "2",
+        "power": "2",
+        "ability": "4",
+        "deploy": "4",
+        "forfeit": "6",
+        "gametext": "While defending a battle, Interrupts may not be played unless they are [Immune to Sense]. Opponent may not target Grogu with weapons unless each of your Mandalorians and non-[Episode I] Jedi present are 'hit.' Immune to attrition < 3.",
+        "lore": "Blank.",
+        "extraText": [
+          "Force-Sensitive"
+        ]
+      },
+      "rulings": [
+        "If an Interrupt is partially immune to Sense, only the immune functions may be played."
+      ],
+      "legacy": false
+    },
+    {
+      "id": 6933,
+      "gempId": "216_30",
+      "side": "Light",
+      "rarity": "C2",
+      "set": "216",
+      "printings": [
+        {
+          "set": "216"
+        }
+      ],
+      "front": {
+        "title": "•••Heavy Infantry Mandalorian",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual16-Light/large/heavyinfantrymandalorian.gif",
+        "type": "Character",
+        "subType": "Alien",
+        "uniqueness": "***",
+        "destiny": "3",
+        "power": "3",
+        "ability": "2",
+        "armor": "4",
+        "deploy": "3",
+        "forfeit": "4",
+        "icons": [
+          "Warrior",
+          "Permanent Weapon"
+        ],
+        "characteristics": [
+          "Mandalorian"
+        ],
+        "gametext": "May 'fly' (landspeed = 3). May move as a 'react.' Permanent weapon is Flamethrower (may target a character or creature for free; draw destiny; target hit if destiny +1 > defense value).",
+        "lore": "Blank."
+      },
+      "rulings": [
+        "This card's text allows it to move to a battle as a react, not away from a battle as a react."
+      ],
+      "legacy": false
+    },
+    {
+      "id": 6934,
+      "gempId": "216_31",
+      "side": "Light",
+      "rarity": "C",
+      "set": "216",
+      "printings": [
+        {
+          "set": "216"
+        }
+      ],
+      "front": {
+        "title": "•Jedi Holocron",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual16-Light/large/jediholocron.gif",
+        "type": "Device",
+        "uniqueness": "*",
+        "destiny": "2",
+        "gametext": "Deploy on your character of ability > 4. While present: adds 1 to training destiny draws and Force drains here; the first Force lost to a Force drain here is stacked here face down; opponent’s ability required to draw battle destiny here is +1 for each card stacked here.",
```

Comment:
Please use a single ' mark instead of the curly tick mark.

Reply:
I replaced the smartquote in Light.json.
I also added a workflow to automatically fix smartquotes going forward.